style.css

```diff
@@ -0,0 +1,92 @@
+@font-face{
+    font-family: 'marvel-regular';
+    src: url(marvel-regular.ttf);
+}
+
+html {
+    font-size: 10px;
+    background-color: #000000;
+    color: #ffffff;
+    font-family: 'Roboto Mono', monospace;
+    background-image: url("marvel-background-1.png");
+}
+
+h1 {
+    font-family: 'marvel-regular';
+    color: #ffffff;
+    background-color: #b13536;
+    height: 300px;
+    width: 800px;
+    border-color: #ffffff;
+    border-width: 10px;
+    border-style: double;
+    font-size: 15rem;
+    text-align: center;
+    padding: 20px;
+    margin: 0;
+}
+
+header {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    padding: 20px;
+    margin: 0;
+}
+
+main {
+    display: flex;
+    justify-content: center;
+    flex-wrap: wrap;
+    gap: 30px;
+    font-size: 2rem;
+}
+
+.containerItem {
```

Comment:
better to name this something that indicates this is an individual marvel character